### DISCUSSION TITLE
Kata4 Ejercicio 2

### DISCUSSION THREAD

**Discussion by AndresAVargas** [LaunchX-InnovaccionVirtual/CursoIntroPython#50] [Q&A]
Hola buenas tardes compañeros , me quede atorado en la kata 4 al escribir el titulo en el ejercicio 2 
![image](https://user-images.githubusercontent.com/88987587/153293423-20c6fd14-0c02-49eb-8dd1-76708f1a1cdc.png)
me dice que name no esta definido y segun yo si por la parte de arriba 
![image](https://user-images.githubusercontent.com/88987587/153293605-7af19501-1243-4e20-bbcf-c10fec9a6880.png)
otra duda, que datos usamos los de la luna y tierra o los de marte y ganimedes?

**Comment by AzaelBarbosa** [comment]
BUen dia, no lo has definido, dentro de tu cuadro de codigo no existe la variable name

`name = "HOLA"`

Y al menos yo hice el ejercicio con ambos datos.

**Comment by JoseEGJasso** [comment]
Ejecuta el bloque de código donde definiste la variable antes de usarla en otro bloque :)

![Captura de pantalla 2022-02-09 153844](https://user-images.githubusercontent.com/41630679/153294901-31de5a02-aba4-42b8-a60a-7438ce2df6ab.png)

En cuanto a tu otra duda, creo que se usan ambos datos primero los de la luna y la tierra y después los de marte y ganimedes para probar que tu cadena funcione con cualquier dato que se asigne a las variables

**Reply by AndresAVargas** [reply]
tenias razón, no ejecute las variables graciaas C:

**Reply by JoseEGJasso** [reply]
por nada :)
